client-app/webpack.config.js

```diff
@@ -1,13 +1,16 @@
 const configureWebpack = require('@xh/hoist-dev-utils/configureWebpack');
 
 module.exports = (env = {}) => {
-    return configureWebpack({
-        appCode: 'toolbox',
-        appName: 'Toolbox',
-        appVersion: env.appVersion || '2.0-SNAPSHOT',
-        agGridLicenseKey: 'Extreme_Heavy_Industries__Scout_3Devs3_April_2020__MTU4NTg2ODQwMDAwMA==8f836aa25989abb0db36231c347dd436',
-        favicon: './public/favicon.png',
-        devServerOpenPage: 'app/',
-        ...env
-    });
+    return {
+        ...configureWebpack({
+            appCode: 'toolbox',
+            appName: 'Toolbox',
+            appVersion: env.appVersion || '2.0-SNAPSHOT',
+            agGridLicenseKey: 'Extreme_Heavy_Industries__Scout_3Devs3_April_2020__MTU4NTg2ODQwMDAwMA==8f836aa25989abb0db36231c347dd436',
+            favicon: './public/favicon.png',
+            devServerOpenPage: 'app/',
+            ...env
+        }),
+        node: {fs: 'empty'}
```

Comment:
This is, admittedly, a bit of a hack. It exists because babel tries to import 'fs' (but not actually use it), and this turns that into a no-op.

Reply:
This is a pretty standard approach we have used on client projects for the same reason - we could consider baking it into hosit-dev-utils.